.github/workflows/deploy-to-aks.yaml

```
@@ -20,7 +21,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

suggestion: Same as above
```suggestion
        uses: actions/checkout@v5
```